tests/project/goost/core/types/test_graph.gd

```diff
@@ -246,8 +246,8 @@ func test_self_loop():
     var a = graph.add_vertex("a")
     var b = graph.add_vertex("b")
 
-    graph.add_edge(a, a)
-    graph.add_edge(b, a)
+    var _e1 = graph.add_edge(a, a)
```

Comment:
Why though?

Reply:
This is a way to silence a warning about unused variables. I've enabled "treat warnings as errors" for tests.

Reply:
Ohh alright, thanks for your response!